### PR TITLE
Block adder

### PR DESCRIPTION
**Pull request by catcosmo**

### CODE REVIEW

File: froide/foirequest/templates/foirequest/show.html [hunk 1]
```diff
@@ -513,11 +517,12 @@
 
   <div class="col-lg-3 col-md-3">
     <ul class="nav nav-compact hidden-print">
-      <li class="nav-header">{% trans "Followers" %}</li>
+       <li class="nav-header">{% trans "Followers" %}</li>
       {% with count=object.follow_count %}
-        <li>
+        {% block followers %}<li>{% endblock followers %}
```
Comment (fin) [inline froide/foirequest/templates/foirequest/show.html]: this doesn't seem right

File: froide/templates/index.html [hunk 2]
```diff
@@ -23,6 +23,8 @@
         <label for="index-search">
           {% blocktrans %}Search {{ foicount }} requests and {{ pbcount }} public bodies:{% endblocktrans %}
         </label>
+      {% block button %}
```
Comment (fin) [inline froide/templates/index.html]: could you rename this to something more descriptive?

**Comment by fin** [comment]
thanks for the pull request!

i'll test this with my deployment, but everything seems fine from reading the code.

some requests: could you remove the .idea folder and the "{%" file? seems like they ended up in the PR by mistake

please also check out the "followers" block (only a list item seems weird, did you miss some stuff) and maybe rename the "button" block to something more descriptive (sendrequest-button or something?)

**Comment by catcosmo** [comment]
thanks for checking!
did adjust that.

**Comment by stefanw** [comment]
Cleaned out .idea files and squash merged in 9448460a674cbab022580f4c06822fa86cd39f28.

**Pull request closed without merging**